### PR TITLE
chore/logs: impl Debug and add info logs

### PR DESCRIPTION
**Pull request by Yoga07**

### CODE REVIEW

File: src/node/mod.rs [hunk 1]
```diff
@@ -152,9 +158,19 @@ impl<R: CryptoRng + Rng> Node<R> {
...
+            RunAsAdult(duty) => {
+                info!("Handling Adult duty: {:?}", duty);
+                self.duties.adult_duties()?.process(&duty)
+            }
+            RunAsElder(duty) => {
+                info!("Handling Elder duty: {:?}", duty);
+                self.duties.elder_duties()?.process(duty)
+            }
+            RunAsNode(duty) => {
+                info!("Handling Adult duty: {:?}", duty);
```
Comment (oetyng) [inline src/node/mod.rs:170]: Should be "Handling **Node** duty: {:?}"

File: src/node/mod.rs [hunk 2]
```diff
@@ -114,8 +114,14 @@ impl<R: CryptoRng + Rng> Node<R> {
...
             let result = match self.receiver.next_event() {
-                Received::Client(event) => ProcessClientEvent(event).into(),
-                Received::Network(event) => ProcessNetworkEvent(event).into(),
+                Received::Client(event) => {
+                    info!("Received a Client Event from quic-p2p: {:?}", event);
+                    ProcessClientEvent(event).into()
+                }
+                Received::Network(event) => {
+                    info!("Received a Network Event from routing: {:?}", event);
+                    ProcessNetworkEvent(event).into()
```
Comment (oetyng) [inline src/node/mod.rs:123]: Don't both of these come from `quic-p2p`?

File: src/node/node_ops.rs [hunk 3]
```diff
@@ -162,10 +176,33 @@ impl Into<NodeOperation> for MessagingDuty {
...
+            Self::SendHandshake { address, .. } => write!(
+                f,
+                "SendHandshake [ address: {:?}, response: (...) ]",
+                address
+            ),
+            Self::SendToAdults { targets, msg } => {
+                write!(f, "SendToAdults [ target: {:?}, msg: {:?} ]", targets, msg)
+            }
+            Self::SendToNode(env) => write!(f, "SendToNode [ msg: {:?} ]", env),
+            Self::SendToSection(env) => write!(f, "SendToSection [ msg: {:?} ]", env),
```
Comment (oetyng) [inline src/node/node_ops.rs:194]: I wondered first if you meant `environment` here :) (Then I understood it is `envelope`.) Maybe better call it `msg`?

File: src/node/node_ops.rs [hunk 4]
```diff
@@ -367,6 +410,8 @@ impl Into<NodeOperation> for PaymentDuty {
 /// metadata only, but could include
 /// chunks, and are then relayed to
 /// Adults (i.e. chunk holders).
+///
```
Comment (oetyng) [inline src/node/node_ops.rs:413]: Surplus comment line.

File: src/node/mod.rs (hunk 2 above)
Comment (Yoga07) [reply src/node/mod.rs:123]: Technically yes, but it's internal to routing which handles all Network/Node messages for us. In the case of clients, it's a first-hand direct connection between them and us.

**Pull request merged**